### ISSUE TITLE
Pre-trained file cannot be loaded with pytorch 1.0.1

### ISSUE THREAD

**Issue by deism** [Kolkir/mlcpp#11]
I failed to load pre-trained file with latest version of pytorch, is there any scripts converting the file? thanks!

**Comment by Kolkir** [comment]
Could you please provide more details, normally files exported from pytorch can't be directly loaded to with C++ API. Please look at the StateLoader class, there are details how to export from python pytorch.

**Comment by deism** [comment]
Thanks for replying.
The details as follows,
1. I compiled mask_rcnn_pytorch in VS 2017 x64 on Pytorch 1.0.1 for windows downloaded from pytorch.org

2. I download 3 pre-trained files from the addresses listed in readme(https://drive.google.com/file/d/1Y9Qq4u-DxiM_eYJ7w4-VXWp4XmUeoU3t/view?usp=sharing...)

3. I failed to load the pre-trained files, because they were created by pytorch 1.0 preiview and pytorch 1.01 does not take them, Actually if there is "TORCH1" in the head of file, it will be rejected

4. I tried to convert  mask_rcnn_cocol.pth to C++ compatible in project https://github.com/multimodallearning/pytorch-mask-rcnn, with pytorch 1.01, unfortunately there is conflicts in model. The code as below,
model.load_state_dict(torch.load(COCO_MODEL_PATH))
example = torch.rand(1, 3, 224, 224)
traced_script_module = torch.jit.trace(model, example)
traced_script_module.save("D:/model.pt")

Any suggestions? thanks!

**Comment by Kolkir** [comment]
Please use this one [ https://github.com/mjstevens777/pytorch-mask-rcnn]( https://github.com/mjstevens777/pytorch-mask-rcnn/tree/feat/build) python implementation because original from multimodallearning is outdated.

**Comment by deism** [comment]
Please check this guy's job at your convenience. https://github.com/darolt/mask_rcnn

**Comment by Kolkir** [comment]
I've update parameters [file](https://drive.google.com/file/d/1H8_0uxCt7J7QIqQWs2QL-fW558-jRm9a/view?usp=sharing) to be compatible with PyTorch v1.0.1.
